python/LLM Examples/secure_prompt_from_file.py

```diff
@@ -0,0 +1,71 @@
+from privateai_client import PAIClient
+from privateai_client import request_objects
+import openai
+import sys
+import demo_config
+import argparse
+import json
+import base64
+import os
+
+# Initialize parser
+parser = argparse.ArgumentParser(description="Secure LLM prompting from a file")
+parser.add_argument("-d", "--directory", required=True, help="directory")
+parser.add_argument("-f", "--file", required=True, help="File to redact")
+parser.add_argument("-t", "--filetype", required=True, help="file type")
```

Comment:
I would actually only accept the extension itself here (eg. pdf). I'll put a sample below on how to validate it for the deid service so we can make it a tad more robust and won't have to split anything. 
The help could also be updated to show a sample like pdf or whatever just for clarity